packages/atomic/src/components/formats/format-common.ts

```diff
@@ -0,0 +1,23 @@
+import {buildCustomEvent} from '../../utils/event-utils';
+
+export type NumberFormatter = (
+  value: number | string,
+  languages: string[]
+) => string;
+
+export const dispatchNumberFormatEvent = (
+  formatter: NumberFormatter,
+  element: Element
+) => {
+  const event = buildCustomEvent('atomic/numberFormat', formatter);
```

Comment:
Any component that will listen to that event can use the formats web component as children

Reply:
This might pose a small challenge for documentation, to be able to express that component X should be used with component Y (child/parent).

Food for thoughts @dlutzCoveo :)

Reply:
RIght now only that facet, in the future, possibly some result template component e.g. `atomic-result-number`

Reply:
We could leverage doc tags here 🤔